### DISCUSSION TITLE
Scanning a remote repository with an access token

### DISCUSSION THREAD

**Discussion by q00td** [zricethezav/gitleaks#1133] [Q&A]
Hello !

I'm trying to scan a remote repo with an access token.
I'v built gitleaks from the source but when i tried to use --repo-url and --access-token (as i'v seen example online), i get an error unknow flags.

I tried to see if the arguments have changed or not but with no luck.

Could anyone tell me if it is still possible or if it is an alternative version.

Regards

**Comment by amithm7** [comment]
`--repo-url` got replaced with `--repo` in v7. See https://github.com/gitleaks/gitleaks/pull/469

[v8.21.2](https://github.com/gitleaks/gitleaks/tree/v8.21.2) doesn't doesn't seem to have `--repo` option either

Maybe it got removed in [v8](https://github.com/gitleaks/gitleaks/issues/683), as it mentions "Removed cloning support"
